mlflow/pytorch/__init__.py

```diff
@@ -366,7 +427,10 @@ def _load_model(path, **kwargs):
         pickle_module_path = os.path.join(path, _PICKLE_MODULE_INFO_FILE_NAME)
         with open(pickle_module_path, "r") as f:
             pickle_module_name = f.read()
-        if "pickle_module" in kwargs and kwargs["pickle_module"].__name__ != pickle_module_name:
+        if (
```

Comment:
please consider putting formatting changes into separate PR, because they increase the scope of the review.

Reply:
sure i will send a separate PR for formatting changes. Thanks.